### ISSUE TITLE
Scroller margin is ignored

### ISSUE THREAD

**Issue by kozuch** [cubiq/iscroll#766]
If the scroller DIV has top or bottom margin, this is ignored causing wrong scrolling height on version 5.1.2. Probably similar to this issue:
https://github.com/cubiq/iscroll/issues/32

**Comment by vanuatu** [comment]
Same issue here!

**Comment by datoml** [comment]
I can't recommend this fix because its not working with dynamic content.
After calling the refresh function the calculation of the height will be wrong.

```Javascript
var marginY = this.scroller.offsetParent ? (this.scroller.offsetParent.scrollHeight - this.scroller.offsetHeight) : 0;
    this.scrollerHeight =this.scroller.offsetHeight + marginY;
```

**Comment by luin** [comment]
Same issue here. Setting the scroller to `overflow: hidden` to trigger BFC works for me though.

**Comment by alvarotrigo** [comment]
I wonder why iScrollj.s doens't just add `overflow:hidden` by default on the scroller element.

**Comment by glebmachine** [comment]
@alvarotrigo Because sometimes you need to have `overflow: visible` by design

**Comment by alvarotrigo** [comment]
@glebmachine you mean, if the developer wants something to overflow?

**Comment by glebmachine** [comment]
Yes, like pages list in facebook app: http://joxi.ru/48Anqa0IXY872O.jpg
In this example canvas with data overflowing layout of scrolling component

**Comment by alvarotrigo** [comment]
So I guess this issue is definitely a bug in iScroll.js? As the docs doesn't seem to state anything about `overflow:hidden` in the scroller?

**Comment by kozuch** [comment]
Anyone going to fix this? Or is it fixed in v5.1.3 or v5.2.0?
